plugins/grpc/csharp/v1.65.0/buf.plugin.yaml

```diff
@@ -9,3 +9,11 @@ output_languages:
   - csharp
 spdx_license_id: Apache-2.0
 license_url: https://github.com/grpc/grpc/blob/v1.65.0/LICENSE
+registry:
+  nuget:
+    target_frameworks:
+      - netstandard2.0
+    deps:
+      - name: Grpc.Net.Common
+        version: 2.63.0
```

Comment:
Unclear how to determine which version of these packages is required by the protoc plugin, so I'm starting w/ the latest stable version here.